### ISSUE TITLE
Нужна помощь с модальными окнами

### ISSUE THREAD

**Issue by ddiyanov** [ivannin/simple-basket#3]
Большое спасибо за Ваш труд!
У меня в процессе интеграции макета возникли сложности с реализацией корзины в модальном окне. Возможно ли такое реализовать? 
Сайт лендинг, одна страница.  
Я разместил шорткод корзины на главной странице. 
В настройках "путь к корзине" указал главную страницу. 
Контент главной страницы вывожу в модальном окне. 
Все работает но не хватает перезагрузки страницы, чтоб обновлялась информация в корзине (модальном окне). Если Вас не затруднит, подскажите как можно это реализовать?

**Comment by ivannin** [comment]
Нет, наверное не подскажу. Мы много лет уже не поддерживаем этот проект, и в нем нет функций управления выводом. Он просто показывает корзину. 

Мой вам совет -- не изобретайте велосипед. Ставьте WooCommerce и реализуйте задачу в нем
